app/services/course/assessment/programming_evaluation_service.rb

```diff
@@ -107,7 +108,7 @@ def execute
   def create_evaluation
     Course::Assessment::ProgrammingEvaluation.create(
       course: @course, language: @language, package_path: @package, memory_limit: @memory_limit,
-      time_limit: @time_limit.to_i
+      time_limit: @time_limit
```

Comment:
I tried it manually and it works. Will check it again with byebug. Let you know when I've double checked.